### DISCUSSION TITLE
Improving project management.

### DISCUSSION THREAD

**Discussion by prjctimg** [prjctimg/huetiful#81] [Q&A]
This discussion started [here](https://github.com/prjctimg/huetiful/discussions/62#discussioncomment-7483061) when @foss2cyber and @adarshswaminath had a collision on the same issue. 

To help reduce such scenarios, can we come up with a way to make sure that a contributor has confidence on what they're doing i.e getting an issue they're interested in assigned to them. 

@takanome-dev suggested creating a CONTRIBUTING file so that the contribution guidelines are clearly defined for everyone!

That being said, I think a project roadmap is essential to keep everyone motivated. So to help keep everyone updated, I'll open a new discussion where contributors can pitch in ideas on what features we should implement next. As we're approaching v2, we may have to shift our focus to making the library include more edge use cases with APIs such as WebGL (is it possible ?🤔) et cetera. 

Let's hear your suggestions on the best ways of keeping our project tidy and manageable!

**Comment by adarshswaminath** [comment]
Hi @prjctimg 

@foss2cyber, it would be great if you could take over the issue. Unfortunately, I'm unable to work on it at the moment. Thanks for your help!"

**Reply by foss2cyber** [reply]
@adarshswaminath Sorry for repyling so late, but I was unfortunately held up elsewhere. I'll definitely work on this now.

**Reply by prjctimg** [reply]
> @adarshswaminath Sorry for repyling so late, but I was unfortunately held up elsewhere. I'll definitely work on this now.

@foss2cyber Pardon the late response. Network issues in the area.

What's the state of our documentation site? Should we add more features to the Library or should we focus all efforts on the docs for now ? 

Just released v1.7.5 now

**Reply by foss2cyber** [reply]
@prjctimg Sorry for repyling sooo late. Been having problems with GitHub for some days now. Issues with the ISP blocking 'raw.githubusercontent.com' for some (really annoying) reason. We can focus on the docs if everyone's on board. I'm all for it.

**Reply by prjctimg** [reply]
@foss2cyber No worries! Technical drawbacks are part of the field 😂.


At the moment I'm updating the docs source (.mdx) files to add more explanations and pages since the project hierarchy has broken down. As we approach v1.8.0, I wish to shift the focus to making the project's utilities compliant to W3C standards to make our API truly acceessible and robust. So my hope is that we have a stable documentation site by then. 

I removed the annoying ESLint checks that failed builds for petty reasons e.g using single quotes instead of double quotes.

At the moment, I'm on a dual core PC so running the development server is a non starter 😆 so I won't be very useful for a while. We could make small incremental changes towards the desired goal. The issue tracker already has issues open for these suggested changes.
